jupyter-R.el

```diff
@@ -11,12 +11,17 @@
         (t (cl-call-next-method))))
 
 (defun jupyter-org-result--R-html (data)
+  "Parse and convert DATA of type 'text/html' into a prettier form.
+If DATA is not a <table>, <ol>, or <dl>, returns nil."
```

Comment:
We should also include `<ul>` here, right?

Reply:
I so far haven't seen the R kernel output `<ul>`, but I've added it in case.